rasa/shared/utils/schemas/events.py

```diff
@@ -0,0 +1,155 @@
+ENTITIES_SCHEMA = {
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "start": {"type": "integer"},
+            "end": {"type": "integer"},
+            "entity": {"type": "string"},
+            "confidence": {"type": "number"},
+            "extractor": {"type": ["string", "null"]},
+            "value": {},
+            "role": {"type": ["string", "null"]},
+            "group": {"type": ["string", "null"]},
+        },
+    },
+}
+
+INTENT = {
+    "type": "object",
+    "properties": {"name": {"type": "string"}, "confidence": {"type": "number"}},
+}
+
+EVENTS_SCHEMA = {
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "event": {"type": "string"},
+            "timestamp": {"type": ["number", "null"]},
+            "metadata": {"type": ["object", "null"]},
+        },
+        "required": ["event"],
+        "oneOf": [
+            {
+                "properties": {
+                    "event": {"const": "user"},
+                    "text": {"type": ["string", "null"]},
+                    "input_channel": {"type": ["string", "null"]},
+                    "message_id": {"type": ["string", "null"]},
+                    "parse_data": {
+                        "type": "object",
+                        "properties": {
+                            "text": {"type": "string"},
+                            "intent_ranking": {"type": "array", "items": INTENT},
+                            "intent": INTENT,
+                            "entities": ENTITIES_SCHEMA,
+                            "response_selector": {
+                                "type": "object",
+                                "properties": {
+                                    "all_retrieval_intents": {"type": "array"},
```

Comment:
array of what?

Reply:
I wasn't sure myself where to look that info up since the example you provided above was an empty array:
```
"response_selector": {
          "all_retrieval_intents": [],
```

Reply:
You can train a model (e.g. one from the example dir) and then run with `rasa run --enable-api`. You can then
* speak to the model via the REST webhook
* retrieve the tracker for the conversation so that you can see all events

I'm using [Insomnia](https://insomnia.rest) for my REST requests. Happy to give you an introduction and / or share my request collection with you

Reply:
I've just had an attempt at this using Postman, managed to do step 2 (get the tracker with initial events which are`ActionExecuted` (action_session_start) and  `SessionStarted`), but i don't know how to do step 1 - speak to the bot so I can send an UserUttered event through the `responseselectorbot` 😬